examples/converter.php

```diff
@@ -31,8 +31,7 @@
         }
     }
 
-    // Stupid PHP :(
-    if (get_magic_quotes_gpc() and isset($_REQUEST['data'])) {
```

Comment:
I can't remember why I need to add `get_magic_quotes_gpc()` here. But I think it had something to do with the differences between PHP CLI and Apache.

Any ideas?

Reply:
Its in an example, so I don't really care for now. A revert is fine with me.

PHP doc says:
> This function has been DEPRECATED as of PHP 7.4.0. Relying on this function is highly discouraged.

https://www.php.net/manual/en/function.get-magic-quotes-gpc.php

Reply:
It is true my _"Stupid PHP"_ comment is not very helpful. But here is the original commit with the commit message _"Fix for if Magic Quotes are turned on"_:
f01478cfae3327953c0114854d80b8b3f01a3d7f

Reply:
Example wont work in PHP 8.0 because there is no `get_magic_quotes_gpc` anymore.

Reply:
Anyway magic quotes feature has been removed from PHP in 5.4.0 (see [here](https://www.php.net/manual/en/security.magicquotes.php)) so in any PHP version EasyRdf is supporting (and any PHP version which is still supported out there) `get_magic_quotes_gpc() and isset($_REQUEST['data'])` **always returns `false`** making whatever is inside this `if` an unreachable statement.

By the way in PHP 8 also the `get_magic_quotes_gpc()` has been removed.

Reply:
Thanks @zozlak. Really useful to know that the feature was dropped in PHP 5.4.0 (which is no-longer supported by EasyRdf). 

That means the whole if statement should be deleted, not just the call to `get_magic_quotes_gpc()`.